src/routes/_surfaces/workspace/explore/leaderboards/Leaderboard.svelte

```diff
@@ -103,7 +113,11 @@
 
   /** replace data after fetched. */
   $: if (!$topListQuery?.isFetching) {
```

Comment:
On using `console.log($topListQuery?.data);` I am getting measure name as `undefined`. This is happening for all the dashboards and dimensions 
<img width="446" alt="Screenshot 2022-09-08 at 5 36 23 PM" src="https://user-images.githubusercontent.com/4402679/189117880-8b95fe5c-2407-4392-b4fa-0c73d7faa279.png">

Also the `meta` tag is empty.  The column name with type is required for displaying the data in the virtualized table

```
    "meta": [
      {
        "name": "dim-name",
        "type": "VARCHAR"
      },
      {
        "name": "m1",
        "type": "BIGINT"
      },
      {
        "name": "m2",
        "type": "DOUBLE"
      },
```

`PreviewTable` component gets this metadata from `currentDerivedSource?.profile`. Probably that could be reused here?